diff --git a/node_modules/@0x5e/react-native-alipay/android/build.gradle b/node_modules/@0x5e/react-native-alipay/android/build.gradle
index 9a20c67..9418aa7 100644
--- a/node_modules/@0x5e/react-native-alipay/android/build.gradle
+++ b/node_modules/@0x5e/react-native-alipay/android/build.gradle
@@ -35,5 +35,8 @@ repositories {
 
 dependencies {
     compile 'com.facebook.react:react-native:+'
-    compile(name: 'alipaySdk-15.7.5-20200422171619', ext: 'aar')
+    // compile(name: 'alipaySdk-15.7.5-20200422171619', ext: 'aar')
+    // utdid冲突，如果集成阿里百川，则需要用不含utdid的包
+    compile files("${rootProject.projectDir.absolutePath}/app/libs/alipaysdk-15.8.03.210428205839.aar")
+    // compile files("${rootProject.projectDir.absolutePath}/app/libs/alipaysdk-noutdid-15.8.03.210428205839.aar")
 }
