diff --git a/node_modules/@kafudev/react-native-alipay/android/build.gradle b/node_modules/@kafudev/react-native-alipay/android/build.gradle
index 33407a0..4869cee 100644
--- a/node_modules/@kafudev/react-native-alipay/android/build.gradle
+++ b/node_modules/@kafudev/react-native-alipay/android/build.gradle
@@ -57,7 +57,7 @@ dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:+"  // From node_modules
 
-    compile files('libs/alipaysdk-15.8.03.210428205839.aar')
+    // compile files('libs/alipaysdk-15.8.03.210428205839.aar')
     // utdid冲突，如果集成阿里百川，则需要用不含utdid的包
-    // compile files('libs/alipaysdk-noutdid-15.8.03.210428205839.aar')
+    compile files('libs/alipaysdk-noutdid-15.8.03.210428205839.aar')
 }
