diff --git a/node_modules/@nozbe/watermelondb/WatermelonDB.podspec b/node_modules/@nozbe/watermelondb/WatermelonDB.podspec
index 1f3af50..4654cd2 100644
--- a/node_modules/@nozbe/watermelondb/WatermelonDB.podspec
+++ b/node_modules/@nozbe/watermelondb/WatermelonDB.podspec
@@ -12,7 +12,8 @@ Pod::Spec.new do |s|
   s.author       = { "author" => package["author"] }
   s.platforms    = { :ios => "9.0", :tvos => "9.0" }
   s.source = { :git => "https://github.com/Nozbe/WatermelonDB.git", :tag => "v#{s.version}" }
-  s.source_files = "native/ios/**/*.{h,m,mm,swift,c,cpp}", "native/shared/**/*.{h,c,cpp}"
+  # s.source_files = "native/ios/**/*.{h,m,mm,swift,c,cpp}", "native/shared/**/*.{h,c,cpp}"
+  s.source_files = "native/ios/WatermelonDB/*.{h,m,mm,swift,c,cpp}", "native/ios/WatermelonDB/SupportingFiles/*.{h,m,mm,swift,c,cpp}", "native/shared/**/*.{h,c,cpp}"
   s.public_header_files = 'native/ios/WatermelonDB/SupportingFiles/Bridging.h'
   s.requires_arc = true
   # simdjson is annoyingly slow without compiler optimization, disable for debugging
