diff --git a/node_modules/@react-native-community/geolocation/android/src/main/java/com/reactnativecommunity/geolocation/BaseLocationManager.java b/node_modules/@react-native-community/geolocation/android/src/main/java/com/reactnativecommunity/geolocation/BaseLocationManager.java
index 72cc08d..f6a36a5 100644
--- a/node_modules/@react-native-community/geolocation/android/src/main/java/com/reactnativecommunity/geolocation/BaseLocationManager.java
+++ b/node_modules/@react-native-community/geolocation/android/src/main/java/com/reactnativecommunity/geolocation/BaseLocationManager.java
@@ -49,8 +49,10 @@ public abstract class BaseLocationManager {
     }
 
     protected static void putIntoMap(WritableMap map, String key, Object value) {
-        if (value instanceof Integer || value instanceof Long) {
-            map.putInt(key, (Integer) value);
+        if (value instanceof Integer) {
+             map.putInt(key, (Integer) value);
+        } else if (value instanceof Long) {
+             map.putInt(key, ((Long) value).intValue());
         } else if (value instanceof Float) {
             map.putDouble(key, (Float) value);
         } else if (value instanceof Double) {
