diff --git a/node_modules/react-native-mmkv/android/build.gradle b/node_modules/react-native-mmkv/android/build.gradle
index 0fd75c2..422ecf1 100644
--- a/node_modules/react-native-mmkv/android/build.gradle
+++ b/node_modules/react-native-mmkv/android/build.gradle
@@ -206,7 +206,7 @@ task downloadBoost(dependsOn: createNativeDepsDirectories, type: Download) {
   def transformedVersion = BOOST_VERSION.replace("_", ".")
   def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
   if (REACT_NATIVE_VERSION < 69) {
-    srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
+    srcUrl = "https://gh.kafudev.com/https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
   }
   src(srcUrl)
   onlyIfNewer(true)
@@ -226,7 +226,7 @@ task prepareBoost(dependsOn: downloadBoost, type: Copy) {
 }
 
 task downloadDoubleConversion(dependsOn: createNativeDepsDirectories, type: Download) {
-  src("https://github.com/google/double-conversion/archive/v${DOUBLE_CONVERSION_VERSION}.tar.gz")
+  src("https://gh.kafudev.com/https://github.com/google/double-conversion/archive/v${DOUBLE_CONVERSION_VERSION}.tar.gz")
   onlyIfNewer(true)
   overwrite(false)
   dest(double_conversion_file)
@@ -242,7 +242,7 @@ task prepareDoubleConversion(dependsOn: downloadDoubleConversion, type: Copy) {
 }
 
 task downloadFolly(dependsOn: createNativeDepsDirectories, type: Download) {
-  src("https://github.com/facebook/folly/archive/v${FOLLY_VERSION}.tar.gz")
+  src("https://gh.kafudev.com/https://github.com/facebook/folly/archive/v${FOLLY_VERSION}.tar.gz")
   onlyIfNewer(true)
   overwrite(false)
   dest(folly_file)
@@ -258,7 +258,7 @@ task prepareFolly(dependsOn: downloadFolly, type: Copy) {
 }
 
 task downloadGlog(dependsOn: createNativeDepsDirectories, type: Download) {
-  src("https://github.com/google/glog/archive/v${GLOG_VERSION}.tar.gz")
+  src("https://gh.kafudev.com/https://github.com/google/glog/archive/v${GLOG_VERSION}.tar.gz")
   onlyIfNewer(true)
   overwrite(false)
   dest(glog_file)
