diff --git a/node_modules/react-native-fast-image/android/build.gradle b/node_modules/react-native-fast-image/android/build.gradle
index cdbdbf3..be9ee29 100644
--- a/node_modules/react-native-fast-image/android/build.gradle
+++ b/node_modules/react-native-fast-image/android/build.gradle
@@ -52,7 +52,7 @@ repositories {
     mavenCentral()
     google()
     maven {
-        url "https://maven.google.com"
+        url "https://maven.aliyun.com/repository/central"
     }
 }
 
