diff --git a/node_modules/react-native/ReactAndroid/build.gradle b/node_modules/react-native/ReactAndroid/build.gradle
index b2d44d1..307c0a4 100644
--- a/node_modules/react-native/ReactAndroid/build.gradle
+++ b/node_modules/react-native/ReactAndroid/build.gradle
@@ -70,7 +70,7 @@ final def prepareBoost = tasks.register("prepareBoost", PrepareBoostTask) {
 }
 
 task downloadDoubleConversion(dependsOn: createNativeDepsDirectories, type: Download) {
-    src("https://github.com/google/double-conversion/archive/v${DOUBLE_CONVERSION_VERSION}.tar.gz")
+    src("https://gh.kafudev.com/https://github.com/google/double-conversion/archive/v${DOUBLE_CONVERSION_VERSION}.tar.gz")
     onlyIfNewer(true)
     overwrite(false)
     dest(new File(downloadsDir, "double-conversion-${DOUBLE_CONVERSION_VERSION}.tar.gz"))
@@ -86,7 +86,7 @@ task prepareDoubleConversion(dependsOn: dependenciesPath ? [] : [downloadDoubleC
 }
 
 task downloadFolly(dependsOn: createNativeDepsDirectories, type: Download) {
-    src("https://github.com/facebook/folly/archive/v${FOLLY_VERSION}.tar.gz")
+    src("https://gh.kafudev.com/https://github.com/facebook/folly/archive/v${FOLLY_VERSION}.tar.gz")
     onlyIfNewer(true)
     overwrite(false)
     dest(new File(downloadsDir, "folly-${FOLLY_VERSION}.tar.gz"))
@@ -102,7 +102,7 @@ task prepareFolly(dependsOn: dependenciesPath ? [] : [downloadFolly], type: Copy
 }
 
 task downloadFmt(dependsOn: createNativeDepsDirectories, type: Download) {
-    src("https://github.com/fmtlib/fmt/archive/${FMT_VERSION}.tar.gz")
+    src("https://gh.kafudev.com/https://github.com/fmtlib/fmt/archive/${FMT_VERSION}.tar.gz")
     onlyIfNewer(true)
     overwrite(false)
     dest(new File(downloadsDir, "fmt-${FMT_VERSION}.tar.gz"))
@@ -118,7 +118,7 @@ task prepareFmt(dependsOn: dependenciesPath ? [] : [downloadFmt], type: Copy) {
 }
 
 task downloadLibevent(dependsOn: createNativeDepsDirectories, type: Download) {
-    src("https://github.com/libevent/libevent/releases/download/release-${LIBEVENT_VERSION}-stable/libevent-${LIBEVENT_VERSION}-stable.tar.gz")
+    src("https://gh.kafudev.com/https://github.com/libevent/libevent/releases/download/release-${LIBEVENT_VERSION}-stable/libevent-${LIBEVENT_VERSION}-stable.tar.gz")
     onlyIfNewer(true)
     overwrite(false)
     dest(new File(downloadsDir, "libevent-${LIBEVENT_VERSION}.tar.gz"))
@@ -133,7 +133,7 @@ final def prepareLibevent = tasks.register("prepareLibevent", PrepareLibeventTas
 }
 
 task downloadGlog(dependsOn: createNativeDepsDirectories, type: Download) {
-    src("https://github.com/google/glog/archive/v${GLOG_VERSION}.tar.gz")
+    src("https://gh.kafudev.com/https://github.com/google/glog/archive/v${GLOG_VERSION}.tar.gz")
     onlyIfNewer(true)
     overwrite(false)
     dest(new File(downloadsDir, "glog-${GLOG_VERSION}.tar.gz"))
